core/src/main/java/greencity/controller/UserController.java

```diff
@@ -283,17 +297,17 @@ public ResponseEntity<Long> getActivatedUsersAmount() {
      */
     @Operation(summary = "Update user profile picture")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
-        @ApiResponse(responseCode = "303", description = HttpStatuses.SEE_OTHER),
-        @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
+            @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
+            @ApiResponse(responseCode = "303", description = HttpStatuses.SEE_OTHER),
+            @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
+            @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
     })
     @PatchMapping(path = "/profilePicture")
     public ResponseEntity<HttpStatus> updateUserProfilePicture(
-        @Parameter(description = "pass image as base64") @RequestPart(required = false) String base64,
-        @Parameter(description = "Profile picture") @ImageValidation @RequestPart(required = false) MultipartFile image,
-        @ApiIgnore @AuthenticationPrincipal Principal principal) {
-        String email = principal.getName();
+            @Parameter(description = "pass image as base64") @RequestPart(required = false) String base64,
+            @Parameter(description = "Profile picture") @ImageValidation @RequestPart(required = false) MultipartFile image,
+            @ApiIgnore Authentication authentication) {
+        String email = (String) authentication.getPrincipal();
```

Comment:
Why did you modify this method? How is it going to help fix the bug from the 215 task?